### DISCUSSION TITLE
Delete node type / edge type from `HeteroData` instance.

### DISCUSSION THREAD

**Discussion by ruchajoshi** [pyg-team/pytorch_geometric#7319] [Q&A] [answered]
I am trying to delete an edge type and all its corresponding information from the `HeteroData` instance that was created. I am not sure how can I do that. Can anyone help me with that?

**Comment by rusty1s** [comment, marked as answer]
```python
del data[edge_type]
```
should do the trick.

**Reply by ruchajoshi** [reply]
Thanks, that works.
